resources/dicts/patrols/general/training.json

```diff
@@ -9673,5 +9673,264 @@
+            {
+                "text": "p_l gives an encouraging nod to r_c as the spirit introduces themself and prepares to train with r_c. r_c hesitantly trains with the spirit under p_l's supervision, and {PRONOUN/r_c/subject} {VERB/r_c/come/comes} home with wounds. p_l insists that the training session was an introduction into how real warriors fight.",
+                "exp": 10,
+                "weight": 5,
+                "injury": [
+                    {
+                        "cats": ["r_c"],
+                        "injuries": ["claw-wound", "cat bite"],
+                        "scars": ["TWO", "SIDE", "THROAT"]
+                    }
+                ],
+                "history_text": {
+                    "reg_death": "m_c was killed after training in the Dark Forest.",
+                    "scar": "m_c was scarred after a brutal training session in the Dark Forest."
+                },
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["platonic", "respect"],
+                        "amount": 5
+                    }
+                ]
+            },
+            {
+                "text": "This cat, this <i>spirit</i>, has a commanding presence. s_c softly encourages the apprentice to the battle, but also makes sure that r_c doesn't get too hurt by the spirit. The Dark Forest spirit argues that r_c would never find out how real warriors fight, but p_l retorts that this is r_c's first time, and they can focus on more battle moves later.",
+                "exp": 10,
+                "weight": 20,
+                "injury": [
+                    {
+                        "cats": ["r_c"],
+                        "injuries": ["torn pelt"],
+                        "scars": ["TWO", "SIDE", "THROAT"]
+                    }
+                ],
+		"stat_trait": ["compassionate", "thoughtful", "loving"],
+		"can_have_stat": ["p_l"],
+                "history_text": {
+                    "reg_death": "m_c was killed after training in the Dark Forest.",
+                    "scar": "m_c was scarred after a brutal training session in the Dark Forest."
+                },
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["platonic", "comfort", "respect"],
+                        "amount": 5
+                    }
+                ]
+            }
+        ],
+        "fail_outcomes": [
+            {
+                "text": "r_c says that no, no, {PRONOUN/r_c/subject} {VERB/r_c/don't/doesn't} want to fight <i>that</i>. The spirit is offended, but surprisingly, p_l sympathizes with the apprentice, and brings {PRONOUN/r_c/object} to do other safer training.",
```

Comment:
I'd stat lock this to the more forgiving personality traits

Reply:
Suggestion for not stat locked failure: use the current decline text (and add a lovely relationship drop between p_l and app1)?

Reply:
Suggestion for additional stat locked failure: "app1 says that no, no, {PRONOUN/app1/subject} {VERB/app1/don't/doesn't} want to fight <i>that</i>. p_l scoffs. This is for app1's own good."
HUGE relationship drop in trust and respect. Injuries: mangled tail, torn ear, sore, sprain. Implied that p_l force app1 to fight the spirit regardless. Stat locked to traits such as fierce, bloodthirsty, confident, arrogant, vengeful, etc?